travel/views.py

```diff
@@ -24,6 +25,10 @@ def get_object(self):
 class TripList(ListView):
 	model = Trip
 
+class TripUpdate(UpdateView):
```

Comment:
Is this used somewhere behind the scenes? From here it looks redundant with `trip_edit`, but maybe it is my ignorance of Django class-based views and forms. :)